v2/workloadapi/convenience.go

```diff
@@ -12,48 +12,93 @@ import (
 // FetchX509SVID fetches the default X509-SVID, i.e. the first in the list
 // returned by the Workload API.
 func FetchX509SVID(ctx context.Context, options ...ClientOption) (*x509svid.SVID, error) {
-	panic("not implemented")
+	c, err := New(ctx, options...)
+	if err != nil {
+		return nil, err
+	}
+	defer c.Close()
```

Comment:
golangci-lint is going to complain about all these. If we want to ignore the close error (which i think is ok), we should do something like the following:
```suggestion
	defer func() { _ = c.Close() }
```

Reply:
Interesting, `make lint` hasn't complained about it so far.